Automattic/portkey-android
# Cancel error notification once all issues in frames have been resolved

Closes #308 

Title has it.

To test:

1. apply the gist in [this link](https://gist.github.com/mzorz/b0ee3c388c834feeac607044abc837a6), which will make the save image operation fail the first 3 times it gets called.
2. create a story with just one frame image, and add emoji / text
3. tap publish 
4. Tap on the error notification so you are shown the error handling view
5. now, tap RETRY on the errored view, as many times as needed to resolve the issues (3 times), and check each time whether there's an error notification in the system dashboard (pull down form the top edge of the screen). There should be one after each *unsuccessful* RETRY.
7. once the RETRY button disappears, check there are no error notifications in the system dashboard anymore.

![error_note](https://user-images.githubusercontent.com/6597771/80223461-b4484f00-861e-11ea-9bb2-0a3a91ff490e.gif)
